app/helpers/home_helper.rb

```diff
@@ -12,6 +12,12 @@ def markdown(text)
     remove_p_tags_from(document).html_safe
   end
 
+  def leader_text(name, size, x, y, rotation)
```

Comment:
Hey @davidtrussler can we pair on this to add a BE spec test for this helper please?

Reply:
We can indeed (though this isn't the highest priority right now though to be honest) but would be well worth it at some point.

Reply:
@Tr4pSt3R Thinking about it, I'd like to get this PR merged into the feature branch. Can we look at adding the test after that?

Reply:
@davidtrussler I'll work on adding the test to it now and submit a PR to this branch

Reply:
After having a chat withth @margOnline && @davidtrussler we will leave the testing aspect of this for now since it's no longer the priority. FinCap is now the priority. You can merge this into the feature branch and make a note to add test to this in the future :(